web/js/userTasksReport.js

```diff
@@ -285,12 +348,22 @@ Ext.onReady(function () {
             handler: findTasks,
         }],
     });
+    /* Allow listing of user tasks of other users only for an admin user */
+    if ( admin == "" ) {
+        Ext.getCmp('userLogin').setDisabled(true);
+    }
 
     /* Handler to invoke the search service */
     function findTasks () {
-                var baseParams = {
-                    'userId': userId,
-                };
+                var baseParams = {};
+
+                if (Ext.getCmp('userLogin').baseParams.userId == ""){
+                    // If no value was set in the combobox, default to the logged in user
+                    baseParams.userId = userId;
+                } else {
```

Comment:
I agree this fixes the problem with sending an invalid request, but I still think it's better you set the proper flags to make the "user" combo compulsory, so users always have the feedback of whose tasks they are getting.